dev/requirements.txt

```diff
@@ -3,7 +3,7 @@ pip
 # run-time
 boltons>=23.0.0
 conda>=23.3.1
-libmamba>=1.4.1,!=1.4.2
-libmambapy>=1.4.1,!=1.4.2
+# libmamba>=1.4.1,!=1.4.2
+# libmambapy>=1.4.1
```

Comment:
Ran into an issue here that `libmambapy` is installed via `pip -e .` before this step which puts `libmambapy` in <develop> mode when listed with `conda list`. The issue is that when I run the 

```shell
sudo /opt/conda/condabin/conda install -y -p /opt/conda \
    --file /opt/conda-libmamba-solver-src/dev/requirements.txt \
    --file /opt/conda-libmamba-solver-src/tests/requirements.txt
```

It will override the libmambapy and install libmamba (which I don't want). Is it possible for conda to know that a package is installed in develop mode and already installed? I know this is not the correct approach. @jaimergp any ideas on how I might approach this?

Reply:
All I can think of is to have a separate file for workflows that install libmamba in dev mode, and some conditionals here and there. conda should recognize that the deps are satisfied (even if in dev mode) if the versions are set up correctly, but if you are testing an earlier version it might complain. 

That said, is there a problem with installing a published libmamba with their deps, and then overriding with the development version? It should work too, right? You might need a `conda remove --force ...` command to make sure it's overridden, but I think it should be ok.